doc/themes/the-things-stack/assets/css/theme.scss

```
@@ -248,4 +248,25 @@ details + blockquote {
   }
 }
 
+.search-container {
+  background-image: url(../img/search_bg.png);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  height: 120px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.search-bar {
+  background-color: white;
+  width: 55vw;
+  padding: 20px;
+  border-radius: 5px;
+  filter: drop-shadow(0 2px 10px rgba(0, 0, 0, 0.2));
+  font-size: 16px;
+  left: -1px; //hack for left-justified dropdown
```

```suggestion
  left: -1px; // Hack for left-justified dropdown.
```

https://github.com/TheThingsNetwork/lorawan-stack/blob/develop/DEVELOPMENT.md#comments